control-plane/csi-driver/src/bin/node/fsfreeze/mod.rs

```diff
@@ -0,0 +1,47 @@
+pub(crate) use bin::FsFreezeOpt;
+use csi_driver::limiter::VolumeOpGuard;
+use tokio::process::Command;
+use tonic::Status;
+use tracing::debug;
+
+pub(crate) mod bin;
+
+// Default csi-node binary name.
+const CSI_NODE_BINARY: &str = "csi-node";
+// Volume name arg for fs freeze and unfreeze subcommand.
+const FREEZE_VOLUME_ID_ARG_NAME: &str = "--volume-id";
+
+/// Creates a child process that calls up the csi-node binary to execute FIFREEZE and FITHAW using
+/// the volume id.
+pub async fn fsfreeze(volume_id: &str, freeze_op: FsFreezeOpt) -> Result<(), Status> {
+    let volume_id = volume_id.to_string();
+    let binary_name = std::env::args().next().unwrap_or(CSI_NODE_BINARY.into());
+    tokio::spawn(async move {
+        debug!(volume_id = volume_id, "{freeze_op}");
```

Comment:
same comment as before

Reply:
`debug!(volume.id = volume_id, "{freeze_op}");` Changed to this

Reply:
probably volume.uuid better as I noticed we already use that elsewhere in this csi-node

Reply:
Alright, would change to volume.uuid